client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigationInfoBar.tsx

```diff
@@ -55,7 +56,31 @@ const InvestigationInfoBar: React.FC<Props> = ({ onExitInvestigation }: Props) =
 
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
 
-    React.useEffect(() => { 
+    const noInvestigationError = () => {
+        Swal.fire({
+            icon: 'warning',
+            title: 'נכנסת לעמוד חקירה מבלי לעבור בדף הנחיתה! הנך מועבר לשם',
+            customClass: {
+                title: classes.swalTitle
+            },
+            timer: 1750,
+            showConfirmButton: false
+        });
+
+        timeout(1900).then(() => history.push(landingPageRoute));
+    }
+
+    React.useEffect(() => {
+        if (localStorage.selectedEpidemiologyNumber) {
```

Comment:
Why are you using localStorage?

Reply:
i need a way to send data between my tabs when i choose an investigation, this seemed like the best way. its either this or passing the epidemiology number via url which seemed to me more vulnerable